src/index.jsx

```diff
@@ -1,8 +1,24 @@
-import React from 'react';
-import reactCompat from '@streammedev/react-compat';
-import {FlyoutToggle} from './toggle';
+const React = require('react');
+const PropTypes = require('prop-types');
+const {FlyoutToggle} = require('./toggle');
```

Comment:
Does this need to be eestructured now that you removed the import/exports?

Reply:
Yes it does, thanks!